spec/factories/transcript.rb

```diff
@@ -0,0 +1,12 @@
+# rubocop:disable FactoryBot/StaticAttributeDefinedDynamically
+FactoryBot.define do
+  factory :transcript do
+    uid { Faker::Lorem.characters(10) }
+    title { Faker::Lorem.sentence }
+    description { "Faith Bandler interviewed by Carol" }
+    audio_url { "http://google.com" }
+    collection
+    vendor
+  end
+end
+# rubocop:enable FactoryBot/StaticAttributeDefinedDynamically
```

Comment:
To fix this Rubocop offence, you just need to get rid of `{` & `}` when the content is static :)

Ie.: replace `audio_url { "http://google.com" }` with `audio_url "http://google.com"`